src/shared/project-manager-review/ProjectManagerReview.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { storiesOf } from '@storybook/react';
+import graphql from 'babel-plugin-relay/macro';
+import React from 'react';
+import { useLazyLoadQuery } from 'react-relay/hooks';
+import { ProjectManagerReviewQuery } from './__generated__/ProjectManagerReviewQuery.graphql';
+import { themeDecorator } from 'src/stories/decorators/themeDecorator';
+import { ProjectManagerReview } from './ProjectManagerReview';
+import { relayDecorator } from 'src/stories/decorators';
+
+const query = graphql`
```

Comment:
I think using constant data is better than using graphql

Reply:
maybe in another pr